### DISCUSSION TITLE
How do I use this on a home network?

### DISCUSSION THREAD

**Discussion by DRY411S** [belowaverage-org/SuperGrate#101] [Q&A] [answered]
In the source and destination, I am using local network IP addresses.
On the source, 4 year old win 10 pro, 4 user profiles for the family, everything works well and the stores are built
On the destination, brand new win11, with just the single user needed to install, I'm getting access denied/permission errors from WMI when trying to restore from the source.
I've ensured all firewalls, vpns and av software is disabled.
The error seems to relate to the source user not having permission on the destination.
I've followed the Supergrate instructions about changing permissions, but without luck.

The error message is that I should make sure that user account has administrator access on the destination PC.

**Comment by krisdb2009** [comment]
I was able to make this work in the lab a while back, but if I remember correctly you needed the same credentials on both the source and dest. So if you were running this on the source PC make sure the destination has a user that is admin that is the same username and password as what is running on the source. If that doesn't work, I will spin up a quick lab and try to get this to work and let you know.

**Comment by krisdb2009** [comment, marked as answer]
OK I just did this in the lab, when using Super Grate, make sure you are logged into an account that exists on both machines, and make sure that both accounts on each machine is using the same password and both are in the local administrator group.

If you want, you can skip the whole network schenario entirely and just run Super Grate from an external usb drive, after capturing on the source PC, move the drive to the destination and open Super Grate on the destination PC and run the restore from there.

I will work on updating the documentation to better explain both of these scenarios.

**Comment by DRY411S** [comment]
Thank you for confirming how to make this work over a home network. In the end, before I saw that you had replied, I had used the external USB method you mention, so I can confirm that this works too. I'm glad it boils down to unclear documentation rather than lack of functionality, this is a great tool that will save me hours. Thank you.
